rules/aip0233/response_resource_field.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0233
+
+import (
+	"fmt"
+
+	"github.com/gertd/go-pluralize"
+	"github.com/googleapis/api-linter/lint"
+	"github.com/jhump/protoreflect/desc"
+)
+
+// The Batch Create response message should have resource field.
+var responseResourceField = &lint.MessageRule{
+	Name:   lint.NewRuleName("core", "0233", "response-resource-field"),
+	OnlyIf: isBatchCreateResponseMessage,
+	LintMessage: func(m *desc.MessageDescriptor) []lint.Problem {
+		// the singular form the resource name, the first letter is Capitalized.
+		// Note: Use m.GetName()[11 : len(m.GetName())-9] to retrieve the resource
+		// name from the the batch create response, for example:
+		// "BatchCreateBooksResponse" -> "Books"
+		resourceMsgName := pluralize.NewClient().Singular(m.GetName()[11 : len(m.GetName())-9])
```

Comment:
We don't need to check if the resource pluralize, as it has been checked in another rule.

Reply:
And use `strings.Replace` for more readability.

Reply:
This is same case with the one in request_requests_field.go,
This statement is try to grab the singular form of the resource
To increase readability, change to use
strings.TrimPrefix(strings.TrimSuffix(m.GetName(), "Response"), "BatchCreate")
to get the resource in the batach request.